### PR TITLE
Vue: fix bootstrap variables

### PR DESCRIPTION
**Pull request by pascalgrimaud**
Following https://github.com/jhipster/generator-jhipster/pull/14306

---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) was updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Review by qmonmert — APPROVED** [review]

File: generators/client/templates/vue/src/main/webapp/content/scss/_bootstrap-variables.scss.ejs [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * Make sure not to add !default to values here
 */
 
+<%_ if (clientTheme === 'none') { _%>
```
Comment (Tcharl) [inline generators/client/templates/vue/src/main/webapp/content/scss/_bootstrap-variables.scss.ejs:8]: Can you use `clientThemNone` derived variable starting from now?

Comment (pascalgrimaud) [reply generators/client/templates/vue/src/main/webapp/content/scss/_bootstrap-variables.scss.ejs:8]: Ok, forgot about this

Comment (Tcharl) [reply generators/client/templates/vue/src/main/webapp/content/scss/_bootstrap-variables.scss.ejs:8]: I think that many members will soon hate me.

Comment (pascalgrimaud) [reply generators/client/templates/vue/src/main/webapp/content/scss/_bootstrap-variables.scss.ejs:8]: :-D

Comment (qmonmert) [reply generators/client/templates/vue/src/main/webapp/content/scss/_bootstrap-variables.scss.ejs:8]: 😅

**Pull request merged**